docs/errors/usage_errors.md

```diff
@@ -1238,3 +1238,105 @@ except PydanticUserError as exc_info:
 
 [related specification section]: https://typing.readthedocs.io/en/latest/spec/callables.html#unpack-for-keyword-arguments
 [PEP 692]: https://peps.python.org/pep-0692/
+
+
+
+## Unsupported type for `validate_call` {#validate-call-type}
+
+`validate_call` has some limitations on the types it can validate. This error is raised when you try to use it with an unsupported type. Currently the supported types are:
+`LambdaType, FunctionType, MethodType, functools.partial`. For the case of `functools.partial`, the function being partially applied must be one of the supported types.
+
+
+### `@classmethod`, `@staticmethod`, and `@property`
+
+These decorators cannot be put before `validate_call` because they return a class rather than one of the supported types. The correct way to use them is to put `validate_call` before the decorator.
```

Comment:
```suggestion
These decorators must be put before `validate_call`.
```

I don't know if we should mention the fact that they are classes. It's not documented so could very much be a CPython implementation detail.